### PR TITLE
Refactor and document the calendars helper

### PR DESCRIPTION
**Pull request by tessi**
I stumbled across the calendars helper and couldn't look away...

### CODE REVIEW

File: app/helpers/calendars_helper.rb [hunk 1]
```diff
@@ -27,46 +27,43 @@
...
-             "#{localized_month_name target_month} #{target_year}"
-           else
-             "#{localized_month_name target_month}"
-           end
-
-    link_to_month(name, target_year, target_month, options.merge(:class => 'navigate-left'))
+  ##
+  # Returns a html link to a calendar of the previous month.
+  # params:
+  #   year (int): the current year
```
Comment (myabc) [inline app/helpers/calendars_helper.rb]: @tessi what documentation syntax are you using? Kind of looks like http://tomdoc.org/

Comment (tessi) [inline app/helpers/calendars_helper.rb]: @myabc no particular syntax actually (I just removed some of the not necessary lines of my comments) to make them more like rdoc

Comment (myabc) [inline app/helpers/calendars_helper.rb]: @tessi take a look at [Yard](http://yardoc.org/). It's really awesome.

Comment (tessi) [inline app/helpers/calendars_helper.rb]: @myabc thanks, I made the documentation look Yard'ish

**Pull request merged**